src/components/partial-string-match/PartialStringMatch.tsx

```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 import {escape} from 'underscore';
 
 export interface PartialStringMatchProps {
-    wholeString: string;
+    wholeString: React.ReactNode;

```

Comment:
Now that `PartialStringMatch` can parse react nodes, I think it would be cleaner to parse its children. That being said, we can keep the wholeString prop for retro compatibility, but I think we should parse the children also. What do you think?

Reply:
Yeah, it would be better I think. I'll try to make something